Web/JSAssignment1/html/Registration.html

```diff
@@ -0,0 +1,82 @@
+<HTML>
+<head>
+<title>Javascript Assignment 1</title>
+</head>
+<!-- Link To java Script -->
+<script type="text/javascript" src="../js/Registration.js">
+	
+</script>
+<link rel="stylesheet" href="../css/Registration.css">
+<body>
+	<section>
+		<h2>INFORMATION</h2>
+		<form onSubmit="validate()">
+			<!-- Table For Data To Symmetric -->
+			<table>
+				<tr>
+					<!-- Name Label -->
+					<td>Enter First Name<span id="span"> *</span></td>
+					<td><input type="text" id="firstName" name="firstName"></td>
+				</tr>
+				<tr>
+					<!-- Last Name -->
+					<td>Enter Last Name<span id="span"> *</span></td>
+					<td><input type="text" id="lastName" name="lastName"></td>
+				</tr>
+				<tr>
+					<!-- Password -->
+					<td>Enter Password <span id="span"> *</span></td>
+					<td><input type="password" id="password" name="password"></td>
+				</tr>
+				<tr>
+					<!--  Confirm Password -->
+					<td>Confirm Password<span id="span"> *</span></td>
+					<td><input type="password" id="confirmPassword"
+						name="confirmPassword"></td>
+				</tr>
+				<tr>
+					<!--  Age Field -->
+					<td>Enter Your Age <span id="span"> *</span></td>
+					<td><input type="number" id="age" min="12" max="75" step="any"
+						name="age"></td>
+				</tr>
+				<tr>
+					<!--  State Name -->
+					<td>Select State<span id="span"> *</span></td>
+					<td><Select id="state" required onchange="createCity()"
+						name="state">
+							<option>Select State</option>
+							<option value="Rajasthan">Rajasthan</option>
+							<option value="UP">Uttar Pradesh</option>
+					</Select></td>
+				</tr>
+				<tr>
+					<!-- City Name -->
+					<td>Select City <span id="span"> *</span></td>
+					<td><select id="city" name="city">
+							<option>Select City</option>
+					</select></td>
+				</tr>
+				<tr>
+					<!--  Address -->
+					<td>Enter Your Address <span id="span"> *</span></td>
+					<td><textarea id="addressLine1" rows="3"></textarea></td>
+				</tr>
+				<tr>
+					<!-- conditions checkbox -->
+					<td colspan="2"><input type="checkbox" id="acceptCondition"
+						onChange="activateSubmit()"><label for="acceptCondition">
```

Comment:
You can pass a parameter "this.value" in the activateSubmit(), so that u can avoid getting the DOM element using its Id and then checking the value of it.